themes/edx.org/lms/templates/course_modes/choose.html

```diff
@@ -21,7 +21,7 @@
             } else {
                 title.attr("aria-expanded", "false");
             }
-        }
+        };
```

Comment:
Can we make this change in both places? https://github.com/edx/edx-platform/pull/11503/files#diff-81310268a943607d9094f376f1c1e863R24